axelrod/strategies/titfortat.py

```diff
@@ -636,3 +636,31 @@ def strategy(self, opponent: Player) -> Action:
 
         # Otherwise play previous move
         return self.history[-1]
+
+@FinalTransformer(D, name_prefix=None)
+class Alexei(Player):
+    """
+    Plays similar to Tit-for-Tat, but always defect on last turn.
+    
+    Names:
+
+    - From http://lesswrong.com/lw/7f2/prisoners_dilemma_tournament_results/    
```

Comment:
Yes that's fine but can you move the site to the bibliography and make the `Names` section fit the guidelines: http://axelrod.readthedocs.io/en/stable/tutorials/contributing/strategy/docstrings.html